mono/metadata/icall-def.h

```diff
@@ -1057,8 +1057,10 @@ HANDLES(ICALL(TYPE_1, "internal_from_handle", ves_icall_System_Type_internal_fro
 HANDLES(ICALL(TYPE_2, "internal_from_name", ves_icall_System_Type_internal_from_name))
 
 ICALL_TYPE(TYPEDR, "System.TypedReference", TYPEDR_1)
-ICALL(TYPEDR_1, "InternalToObject",	mono_TypedReference_ToObject)
-ICALL(TYPEDR_2, "MakeTypedReferenceInternal", mono_TypedReference_MakeTypedReferenceInternal)
+HANDLES(ICALL(TYPEDR_1, "InternalToObject",	mono_TypedReference_ToObject))
+
+// Has raw pointers but not easily convertable at this time.
+NOHANDLES(ICALL(TYPEDR_2, "MakeTypedReferenceInternal", mono_TypedReference_MakeTypedReferenceInternal))
```

Comment:
I don't think we should use NOHANDLES on functions that still require our attention.

Reply:
ok yeah I was thinking that tood.

Reply:
I removed the marker, edited the command, and big bonus: mostly converted the function. The `p` variable remains.